lib/active_merchant/billing/credit_card.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Please remove this empty line, it causes rubocop to fail.

Are you running `bundle exec rake test:local` on your branch before submitting it? I see you have added test output to your PR description, but it looks like you have just pasted output from a previous test run because it shows rubocop passing and it shows remote tests when there are no remote tests to run for this branch.

Please run `bundle exec rake test:local` on your branch and include that output in your PR description.

Reply:
Fixed

Finished in 40.794637 seconds.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
5288 tests, 76248 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
129.62 tests/s, 1869.07 assertions/s
Running RuboCop...
Inspecting 746 files